android/src/main/java/com/swmansion/rnscreens/SearchViewFormatter.kt

```diff
@@ -64,4 +65,8 @@ class SearchViewFormatter(var searchView: SearchView) {
             searchEditText?.hint = placeholder
         }
     }
+
+    fun clearText() {
+        searchEditText?.setText(mDefaultText)
```

Comment:
According to the upper comment, let's change this to

```suggestion
        searchEditText?.setText(null)
```